internal/store/horizontalpodautoscaler_test.go

```diff
@@ -80,7 +80,7 @@ func TestHPAStore(t *testing.T) {
 								},
 								Target: autoscaling.MetricTarget{
 									Value:        resourcePtr(resource.MustParse("10")),
-									AverageValue: resourcePtr(resource.MustParse("12")),
+									AverageValue: resourcePtr(resource.MustParse("0.5")),
```

Comment:
Could we also add a test case when `Value` is a float?

Reply:
Test cases have been updated.